.ci/Jenkinsfile

```diff
@@ -114,7 +120,12 @@ pipeline {
             def agents = readYaml(file: '.ci/.jenkins-agents.yml')
             def parallelTasks = [:]
             agents['agents'].each { agent ->
-              parallelTasks["${agent.REPO}"] = generateStepForAgent(repo: "${agent.REPO}", filesType: "gherkin", filesPath: "${agent.FEATURES_PATH}")
+              if (agent.FEATURES_PATH) {
+                def repository = 'apm-agent-' + env.SELECTED_AGENT
+                if (env.SELECTED_AGENT == 'all' || repository == agent.REPO) {
```

Comment:
WDYT?
```suggestion
                if (env.SELECTED_AGENT == 'all' || "apm-agent-${env.SELECTED_AGENT}" == agent.REPO) {
```

Reply:
Simpler! 👍 

I added the variable for readability, but maybe this needs conciseness